File: Python/Product.py

class Product:
    #atribut
    __price = 0
    __idProduct = 0

    #constructor
    def __init__(self):
        self.__price = 0
        self.__idProduct = 0
        
    #set get
    def setidProduct(self, idProduct):
        self.__idProduct = idProduct
    def getidProduct(self):
        return self.__idProduct

    def setPrice(self, price):
        self.__price = price
    def getPrice(self):
        return self.__price

    

File: PHP/Product.php
<?php

// define product class
class Product
{
	//atribut
	private $idProduct;
	private $price;

	//constructor
	public function Product($idProduct = 0, $price = 0)
	{
		$this->idProduct = $idProduct;
		$this->price = $price;
	}

	// getter and setter
	public function setIdProduct($idProduct)
	{
		$this->idProduct = $idProduct;
	}

	public function getIdProduct()
	{
		return $this->idProduct;
	}

	public function setPrice($price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->price;
	}
}